kraken/pod_scenarios/setup.py

```diff
@@ -127,14 +129,20 @@ def container_run(kubeconfig_path,
     return failed_scenarios, scenario_telemetries
 
 
-
 def container_killing_in_pod(cont_scenario, kubecli: KrknKubernetes):
     scenario_name = cont_scenario.get("name", "")
     namespace = cont_scenario.get("namespace", "*")
     label_selector = cont_scenario.get("label_selector", None)
     pod_names = cont_scenario.get("pod_names", [])
     container_name = cont_scenario.get("container_name", "")
-    kill_action = cont_scenario.get("action", "kill 1")
+    kill_action = cont_scenario.get("action", 1)
+    if not isinstance(kill_action, int):
```

Comment:
I dont think this if statement will ever be hit since we are setting a default if kill_action is not set

Reply:
Think this checks if kill_action is an int if set, if not set in the config we go with 1. Thoughts?